servant-hmac-auth.cabal

```diff
@@ -72,7 +72,7 @@ library
                      , bytestring ^>= 0.10 || ^>= 0.11 || ^>= 0.12
                      , case-insensitive ^>= 1.2
                      , containers >= 0.5.7 && < 0.8
-                     , cryptonite >= 0.25 && < 0.31
+                     , crypton >= 0.31 && < 2.0
```

Comment:
Please add 

```yaml
extra-deps:
  - crypton-1.0.1@sha256:f41316fbc6ad878396e476355e27b70ac35c344d74e3eefafe709e03b192be9e,14527
```

to stack configs for ghc 9.0.2 and 9.2.8.

I'm fine with merging this if you manage to make the CI green

Reply:
Will do, just running this locally to make sure the CI commands work